release-notes/opensearch.release-notes-2.5.0.md

```diff
@@ -2,37 +2,57 @@
 
 ## [2.5]
 ### Added
-- Added jackson dependency to server ([#5366](https://github.com/opensearch-project/OpenSearch/pull/5366))
-- Added experimental support for extensions ([#5347](https://github.com/opensearch-project/OpenSearch/pull/5347)), ([#5518](https://github.com/opensearch-project/OpenSearch/pull/5518)), ([#5597](https://github.com/opensearch-project/OpenSearch/pull/5597)), ([#5615](https://github.com/opensearch-project/OpenSearch/pull/5615)))
-- Adding support to register settings dynamically ([#5495](https://github.com/opensearch-project/OpenSearch/pull/5495))
-- Adding auto release workflow ([#5582](https://github.com/opensearch-project/OpenSearch/pull/5582))
+- Prevent deletion of snapshots that are backing searchable snapshot indexes ([#5365](https://github.com/opensearch-project/OpenSearch/pull/5365))
+- Reject bulk requests with invalid actions ([#5384](https://github.com/opensearch-project/OpenSearch/issues/5384))
+- Add max_shard_size parameter for shrink API ([#5229](https://github.com/opensearch-project/OpenSearch/pull/5229))
+- Add jackson dependency to server ([#5366](https://github.com/opensearch-project/OpenSearch/pull/5366))
```

Comment:
I believe this has been reverted, right?

Reply:
Updated.